### ISSUE TITLE
Y Axis Bounds in Multiple Line Charts not working as intended

### ISSUE THREAD

**Issue by D3nn3** [apache/superset#15586]
1. `0` as input for min `Y AXIS BOUNDS` in `Multiple Line Charts` gets ignored, everything else works though (e.g. 5, 1000, -5).
2. Selecting `Y BOUNDS` does nothing.

Didn't find any open issues regarding this. If I missed it and this is a duplicate please disregard and close the issue.

### Expected results

1. With `0` as input in min `Y AXIS BOUNDS` the Y axis in the visualization should start at 0.
2. With `Y BOUNDS` deselected the min and max value shouldn't necessarily be displayed.

### Actual results

1. `0` as input gets completely ignored and has the same effect as leaving `Min` empty.
2. No matter of selected or not, min and max values always get displayed, no matter the values entered.

### Environment

- browser: Firefox 89.0.2 on Ubuntu
- superset version: `apache/superset:1.2.0`

**Comment by junlincc** [comment]
@D3nn3 hi! thanks for reporting. unfortunately Multi-line chart is not being actively being maintained. We offer Mix Time-series chart as alternative which should support Y axis bound. Please let us know.

**Comment by D3nn3** [comment]
Hi @junlincc , thanks for getting back to me so quickly. Unfortunately the Mixed Timeseries Chart doesn't offer the functionality needed.

The current use case is simple but somehow unsolvable it seems 😅 We want to a chart displaying the number of X and additionally the moving average. We did this creating two different line charts and combining these in a Multiple Line Charts visualization. 

Alternatives that don't seem to work:
- Mixed Timeseries Chart doesn't allow advanced analytics.
- Using a line chart as annotation layer in different visualizations also doesn't work because the `chart` drop down is always empty when selecting `time series` as annotation layer type and `line chart` as annotation source. Not sure if it is a bug or not yet.

I feel like this should easily be possible and I'm missing something...

**Comment by junlincc** [comment]
Hi @D3nn3 We will make sure reaching feature parity before deprecating it, so don't worry. We need to allocate resource right in order to have one solid combination chart that support multiple y-aixs, and we decided Mix Times-series is the one. 

> * Mixed Timeseries Chart doesn't allow advanced analytics.
It's in progress. 
@zhaoyongjie let's push your PR forward and see if @xiezhongfu and @stephenLYZ can use your code for the Mixed chart. 

> * Using a line chart as annotation layer in different visualizations
Likely a bug, we will look into it. @villebro 

In short, we will have both annotation and AA work in the new Mixed chart.

**Comment by stephenLYZ** [comment]
Hi @D3nn3  It seems that https://github.com/apache/superset/issues/14998 is very similar to what you mentioned, see if it can meet your needs. Thanks.

**Comment by D3nn3** [comment]
It seems that the fix was for line charts. I checked and line charts work fine but my problem is with the visualization type "**Multiple** Line Charts".

**Comment by junlincc** [comment]
@D3nn3 you are welcome to take the ownership and contribute codes to Multiple Line Charts or any legacy charts, as we are actively maintaining the alternatives which are more extensible. we will provide support and review to your solution. 🙏

**Comment by villebro** [comment]
We should be able to introduce Advanced Analytics into Mixed Timeseries Chart not that it has been added to the new Timeseries chart. Would that solve your problem @D3nn3 ?

**Comment by D3nn3** [comment]
Adding Advanced Analytics to Mixed Time Series Chart would solve it (only if it can be applied to a single query, e.g. only Query A, and not necessarily to both at the same time).

Another (maybe "better" and more flexible) solution would be the following:
- Add Advanced Analytics to Time Series Chart (afaik this has been done or is planned? v1.2.0 doesn't have this though)
- Fix Time Series Annotation Layers. Currently filters applied in a Dashboard regarding time range don't properly propagate.

This would allow us to mix two (and more!) Time Series charts (with optional & individual advanced analytics) or one/multiple Time Series charts and a Mixed Time Series chart (always via Annotation Layers) where e.g. you have a raw count metric and additionally the rolling window in one chart. I think this solution is more flexible and also allows combining more than two different charts (which would be the limitation of just adding Advanced Analytics to Mixed Time Series Charts) as already mentioned before.

Adding Advanced Analytics to Mixed Time Series Charts would allow an additional use cases that my proposed solution wouldn't though. As soon as you want to have Advanced Analytics in a Bar visualization there doesn't seem a way to realize this. But I think this isn't a typical use case (at least my team never had the urge to do something like this) and could be added in a later release (e.g. v1.4.0 or even later, depending on available resources).

**Comment by rusackas** [comment]
Closing this since Mixed Charts (formerly Mixed Time Series) now contains advanced analytics features. If this does not suffice, we can re-open this, or we might just want to open a new Issue with updated context, since this thread is a bit out of date by now.
